schema/organization/Organization.md

```diff
@@ -0,0 +1,152 @@
+  English, Māori </institution>
+  <institution content-type="contact-email">
+  office@otago.ac.nz </institution>
+  <institution content-type="contact-phone">
+  00641234567 </institution>
+</institution-wrap>
+```
+
+Note that `<institution>` in JATS is an adrress class element
+
+```
+<contrib contrib-type="author">
+<name><surname>Smith</surname>
+<given-names>Jane</given-names></name>
+<degrees>Ph D</degrees>
+<aff>The University of Otago</aff>
+<address>
+<institution content-type="edu">The University of Otago</institution>
+<institution content-type="dept">Department of Computer Science </institution>
+<addr-line>362 Leith Street</addr-line>
+<addr-line> Dunedin 9054/addr-line>
+<country>New Zealand</country>
+<email>jane.smith@otago.ac.nz</email>
+</address>
+</contrib>
+```
+
+### OpenDocument
+
+`Organization` is analogous to the Open Document [`<text:organizations>`](http://docs.oasis-open.org/office/v1.2/os/OpenDocument-v1.2-os-part1.html#__RefHeading__1419060_253892949)
+and [`<text:institutions>`](http://docs.oasis-open.org/office/v1.2/os/OpenDocument-v1.2-os-part1.html#__RefHeading__1418948_253892949) attributes.
+
+### HTML5
+
+At the moment there does not seem to be an HTML5 tag or attribute that `Organization` is analogous to. However, it may be possible to [use link types](http://w3c.github.io/html/links.html#sec-link-types). If you are able to add more information about this, please [edit this file](https://github.com/stencila/schema/edit/master/schema/Organization.schema.yaml).
+
+
+### Crossref
+
+`Organization` is analogous to the Crossref Organization element [`<crossref:organization>`](https://data.crossref.org/reports/help/schema_doc/4.4.0/relations_xsd.html#http___www.crossref.org_relations.xsd_organization) which is the name of an organization (as opposed to a person) that contributed to authoring an entity. If multiple organizations authored an entity, each one should be captured in a unique organization element.
+
+### Citation Style Language (CSL)
+
+At the moment there does not seem to be a schema element in CSL that `Organization` is analogous to. If you are able to add more information about this, please [edit this file](https://github.com/stencila/schema/edit/master/schema/Organization.schema.yaml).
+
+
+### ISA framework
+
+The ISA metadata framework does not include a schema analogous to `Organization`. However, the [`person_schema.json`](https://isa-specs.readthedocs.io/en/latest/isajson.html#person-schema-json) includes elements corresponding to the elements in `Organization` in Stencila Schema.
+
+Example:
+
+```json
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title" : "ISA person schema",
+    "type" : "object",
+    "properties" : {
+        "lastName" : { "Smith" },
+        "firstName" : { "Jane" },
+        "email" : { "jane.smith@otago.ac.nz" },
+        "address" : { "362 Leith Street, Dunedin 9054, New Zealand" },
+        "affiliation" : { "The University of Otago" },
+}
```

Comment:
This example is mixing a JSON Schema definition with a JSON object. An example of an object that conforms to the ISA person schema would be:

```json
{
        "lastName" : "Smith",
        "firstName" : "Jane",
        "email" : "jane.smith@otago.ac.nz",
        "address" : "362 Leith Street, Dunedin 9054, New Zealand",
        "affiliation" : "The University of Otago"
}
```

In any case, I don't think that we need this example.